### ISSUE TITLE
Homogenize “Subject” Nomenclature

### ISSUE THREAD

**Issue by tsalo** [bids-standard/bids-2-devel#14]
We are currently using the terms “participant” and “subject” interchangeably. I propose we make the names homogeneous. In order to afford better extendability with preclinical research, the term “subject” would be significantly more apt.
GitHub Issue: https://github.com/bids-standard/bids-specification/issues/384

This change would include renaming the `participants.tsv` file to `subjects.tsv`, and the `participant_id` column in said file to `subject`.

**Original authors**: @TheChymera

**Comment by TheChymera** [comment]
@tsalo thank you for bringing this up. While the urgency from my side (and that of other preclinical researchers using SAMRI) has decreased, I would again recommend this as a worthwhile change.

**Comment by HenkMutsaerts** [comment]
I agree with this point, just brought this up with @Remi-Gau who pointed me to this issue (thanks!).
Though note that some prefer `participant` over `subject` as the latter can be derogatory. I understand that `participant` is tricky with preclinical research, perhaps `sample` would be most neutral (and generalizable) but `subject` would be OK with me at least.

**Comment by yarikoptic** [comment]
In the light of the BEP032 (attn @bids-standard/bep032) for animal ephys `subject` would also be more appropriate.  So I think it is a valid target for the "immediate" BIDS 2.0  with easy "auto upgrade" path.

**Comment by SylvainTakerkart** [comment]
> I agree with this point, just brought this up with @Remi-Gau who pointed me to this issue (thanks!). Though note that some prefer `participant` over `subject` as the latter can be derogatory. I understand that `participant` is tricky with preclinical research, perhaps `sample` would be most neutral (and generalizable) but `subject` would be OK with me at least.

just for info, "sample" is already used in BIDS (for pre-clinical research, i.e to specify that several "samples", e.g tissue slices, can be extracted from the same "subject" (animal)...

**Comment by yarikoptic** [comment]
FTR, another definitely non-participant'y and even overall inanimate use-case came up yesterday in 
- https://github.com/NeurodataWithoutBorders/nwbinspector/pull/431

so there "subject" IMHO still works as (inanimate) "subject of study".
